src/editor/hotkeys.cpp

```diff
@@ -156,20 +163,20 @@ void CodeEditor::keyPressEvent(QKeyEvent *e)
             return;
         }
         if((e->key() == Qt::Key_Plus && e->modifiers() & Qt::ControlModifier)//ctrl & +
```

Comment:
It will be better if you move these "if" statements to separate function. 
for instance:

void Zoom(int value )
{
if((e->key() == Qt::Key_Minus && e->modifiers() & Qt::ControlModifier)
{	        
	this->zoomOut(1);//decrease zoom	            
	currentZoom += value;	           
	setViewportMargins(lineNumberAreaWidth(), 0, 0, 0);// reset text margin in accordance to linecouter change	          
	return;	           
}	

}

void Zoom(-1 );
void Zoom(1);

Reply:
You can add mCurrentZoom  parameter if it needed

Reply:
fixed it